plugin/src/main/java/io/fabric8/maven/plugin/ResourceMojo.java

```diff
@@ -81,6 +81,12 @@
     private File resourceDir;
 
     /**
+     * Should we use the project's compmile-time classpath to scan for additional enrichers/generators?
+     */
+    @Parameter(property = "fabric8.useProjectClasspath", defaultValue = "false")
+    private boolean useProjectClasspath = false;
```

Comment:
See above, wouldn't make it configurable but enable it always (as this was in the design anyway).